packages/material-ui/src/Tab/Tab.d.ts

```diff
@@ -7,13 +7,15 @@ declare const Tab: ExtendButtonBase<{
     disabled?: boolean;
     fullWidth?: boolean;
     icon?: string | React.ReactElement;
+    indicator?: React.ReactNode;
```

Comment:
@sperry94 It doesn't look like this was needed for the demos. What required this change? It's an internal property. We only document public properties in the type declarations.

Reply:
@eps1lon nothing required this change, I saw that prop in the Tabs component and thought it had been missed in the TS file like `wrapped` was. I have opened PR https://github.com/mui-org/material-ui/pull/15143 to address this!